setup.cfg

```diff
@@ -85,7 +85,7 @@ dask =
     dask[dataframe] >= 2021.10.0
 
 spark = 
-    pyspark >= 3.2.0
+    pyspark >= 3.2.0, <3.3.0
```

Comment:
the latest version of pyspark contains breaking changes

Reply:
What breaks with the latest version? Should we also file an issue to unpin this in the future?

Reply:
One unit test breaks:
- `test_series_methods_returning_frame_no_name[sample_series_spark]` raises a `KeyError`
I'll file an issue to investigate further. At an initial glance, it seems like a problem with having `Series.ww.name` set as None.

Reply:
Tracking issue in https://github.com/alteryx/woodwork/issues/1453